src/app/modules/group/components/group-links/group-links.component.html

```diff
@@ -5,8 +5,10 @@
   >
     <strong>{{ item.name }}</strong>
   </a>
-  <span class="separator" *ngIf="!isLast">{{ i < (maxItemsDisplay - 2) || hasMoreItems ? ',' : 'and' }}</span>
-  <span class="more" *ngIf="hasMoreItems && isLast">
-    and {{ moreItemsCount }} more
+  <span class="separator" *ngIf="!isLast">
+    {{ i < (maxItemsDisplay - 2) || (items || []).length > maxItemsDisplay ? ',' : 'and' }}
```

Comment:
And don't know if there are example with only 2 or 3 elements in the list, but I think there is slight error. In English, it should be "group1, group2**, and** group3" (comma and "and").

BTW the "and" should be localized (and if possible, the way to list... as in French there would not be comma before the "and").

Reply:
Fixed